docker.yaml

```diff
@@ -0,0 +1,141 @@
+package:
+  name: docker
+  version: 24.0.6
+  epoch: 0
+  description: A meta package for Docker Engine and Docker CLI
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - docker-cli
+      - dockerd
+
+environment:
+  contents:
+    packages:
+      - bash
+      - btrfs-progs-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - containerd
+      - coreutils
+      - go
+      - libseccomp-dev
+      - libtool
+      - linux-headers
+      - lvm2-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/moby/moby
+      tag: v${{package.version}}
+      expected-commit: 1a7969545d73537545645f5cd2c79b7a77e7d39f
+
+  - runs: |
+      # Repliace the environment setup performed by docker
+      export DOCKER_BUILDTAGS="seccomp"
+
+      ./hack/make.sh dynbinary
+
+  # Independently managed from moby/moby, but contains many utility scripts needed for the official docker images.
```

Comment:
Should this be a subpackage? Or another top-level package? That might make it easier to keep up to date.

Reply:
I included it in here since the repo doesn't publish any tags or releases, so either way it's going to be difficult to auto update. Tying it to this package at least reminds us to check when `docker` (which has tags) bumps.